api/v1beta2/awscluster_types.go

```diff
@@ -229,7 +230,7 @@ type AWSLoadBalancerSpec struct {
 
 	// LoadBalancerType sets the type for a load balancer. The default type is classic.
```

Comment:
Could we add a comment on what each value for this does and what the new value is useful for?